### DISCUSSION TITLE
Best way to batch update options?

### DISCUSSION THREAD

**Discussion by ruida-shen** [swellstores/community#326] [Help] [answered]
Hi there, I have a couple of questions:

1. How do I batch update a list of products? Specifically the options, for example I am creating an online ordering app for a pizza shop, and I added sausage and mushrooms as options, and let say the shop is out of mushrooms, then how do I disable that option for all pizzas?

2. Do you support price modifiers? Like choosing sausage adds 1 dollar to the item.

Thank you for you replies.

**Comment by sags95** [comment, marked as answer]
Hey @ruida-shen, 

1. This might ultimately depend on how you're configuring your product catalog. If the Pizzas are all individual products, each with their own options for addons, then you would have to loop through products, setting the `option` as `active: false`. 
2. Yes you can enable price modifiers by clicking on the option value, then entering a price in the field
<img width="615" alt="Screenshot 2023-08-01 at 12 51 06 PM" src="https://github.com/swellstores/community/assets/9522171/b7960c0f-c760-4c30-958e-7de1076b81ea">

**Reply by ruida-shen** [reply]
Hi Alex, thanks for the reply. This is very helpful.
